docs/cheaha/slurm/slurm_tutorial.md

```diff
@@ -0,0 +1,528 @@
+
+```bash
+#!/bin/bash
+# Declaring Slurm configuration Options and specifying required resources
+...
+# Loading Software/Libraries
+...
+# Running Code
+...
+```
+
+The last portion is running the actual code or software. Here, the computational task or program intended for execution is launched using specific commands and processes, which depends on the software used and overall computational workflow. For more detailed specification, refer to [Slurm job submission](../slurm/submitting_jobs.md). The following sections present practical examples for writing a Slurm batch script to specific use cases, and Prerequisites to start with the tutorial.
+
+## Prerequisites
+
+If you're new to using Unix/Linux commands and bash scripting, we suggest going through the software carpentry lesson, [The Unix Shell](https://swcarpentry.github.io/shell-novice/). Also, we recommend reviewing the [Cheaha Hardware Information](../../cheaha/hardware.md) to help guide you in chosing appropriate partition and resources.
+
+## Example 1: A Simple Slurm Batch Job
+
+Let us start with a simple example to print `hostname` of the node where your job is submitted. You will have to request for the required resources required to run your job using Slurm parameters. To learn more about individual Slurm paramters given in the example, please refer to [Slurm flag and environment variables](../slurm/submitting_jobs.md/#slurm-flags-and-environment-variables) and the official [Slurm documenation](https://slurm.schedmd.com/).
+
+To test this example, copy the below script in a file named `hostname.job`. This job executes the `hostname` command (line 15) on a single node, using one task, one CPU core, 1 gigabyte of memory, with a time limit of 10 minutes. The output and error logs are directed to separate files with names based on their job name and ID (line 11 and 12). For a more detailed understanding of the individual parameters used in this script, please refer to the section on [Simple Batch Job](../slurm/submitting_jobs.md/#a-simple-batch-job). The following script includes comments, marked with `###`, describing their functions. We will utilize this same notating for annotating comments in subsequent examples.
+
+```py linenums="1"
+#!/bin/bash
+
+### Declaring Slurm configuration Options and specifying required resources
+#SBATCH --job-name=hostname ### Name of the job
+#SBATCH --nodes=1           ### Number of Nodes
+#SBATCH --ntasks=1          ### Number of Tasks
+#SBATCH --cpus-per-task=1   ### Number of Tasks per CPU
+#SBATCH --mem=1G            ### Memory required
+#SBATCH --partition=express ### Cheaha Partition
+#SBATCH --time=00:10:00     ### Estimated Time of Completion
```

Comment:
`### Estimated Time of Completion, 10 minutes`

Reply:
Incorporated the changes